KsApi/lib/ApiDateProtocol.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+public protocol ApiDateProtocol {
```

Comment:
We have something similar already in the project but I'm assuming this needed to be made because of target membership? Also, is this specifically for the cookie expiration?

Reply:
Yeah ideally we'd at some point move things like this into a shared _lib_ or something but right now in order to write that test in KsApi this was necessary. If we moved `DateProtocol` to `KsApi` we'd have to import `KsApi` everywhere that we use it and for now this is just for the one test. Not ideal 🤷‍♂️